package.json

```diff
@@ -19,6 +19,10 @@
     "@angular/platform-browser": "~8.2.14",
     "@angular/platform-browser-dynamic": "~8.2.14",
     "@angular/router": "~8.2.14",
+    "body-parser": "^1.19.0",
+    "bulma": "^0.8.0",
```

Comment:
What's bulma for? some default CSS?

Reply:
CSS package added by Lucas. Needed and utilized in index.html.

Reply:
Bulma provides a ton of useful features. Content scaling is a nice one that I like, also it allows you to style inside the html files instead of the separate scss file. You'll see it when you look at the class="... has-background-grey"